opensearch/src/main/java/org/opensearch/sql/opensearch/storage/OpenSearchIndexScan.java

```diff
@@ -158,6 +160,22 @@ public void pushDownLimit(Integer limit, Integer offset) {
     sourceBuilder.from(offset).size(limit);
   }
 
+  /**
+   * Add highlight to DSL requests.
+   * @param field name of the field to highlight
+   */
+  public void pushDownHighlight(String field) {
+    SearchSourceBuilder sourceBuilder = request.getSourceBuilder();
+    if (sourceBuilder.highlighter() != null) {
+      sourceBuilder.highlighter().field(StringUtils.unquoteText(field));
```

Comment:
Will there be any issues using `StringUtils.unquoteText` with double quotes acting as escape characters? Not sure if your desired output is just unquoting or also the additional functionality unquote provides.

Reply:
The escaping character functionality is not needed for the highlight expression. We simply pass the string to opensearch to receive the response and need a regular quoted string.